examples/hyperspace-roll/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 
 [dependencies]
 agb = { version = "0.18.1", path = "../../agb" }
+portable-atomic = { version = "1.6.0", default-features = false }
```

Comment:
I do wonder if agb should re-export the portable-atomic stuff?

Reply:
I considered it, but I would only do it because our API is not stable.